.github/workflows/vercel-preview.yml

```
@@ -0,0 +1,24 @@
+        uses: amondnet/vercel-action@v25
+        with:
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
+          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
+          working-directory: .
+          vercel-args: "--prod"
+        env:
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+      - name: Output preview URL
+        if: outputs.preview-url != ''
+        run: echo "Preview URL: ${{ steps.vercel-action.outputs.preview-url }}"
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)